### PR TITLE
quotes typos + description repetition

### PR DESCRIPTION
**Pull request by shaqsingh**

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
   <meta property="og:title" content="{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}"/>
   <meta property="og:type" content="article"/>
   <meta property="og:url" content="{{ site.production_url }}{{ page.url | replace:'index.html',''}}"/>
-  <meta property="og:description" content="{% if page.description %}{{ page.description }} | {{ page.subtitle }}{% else %}{{ site.description }}{% endif %}" />
```
Comment (dcwalk) [inline _includes/head.html:21]: We shouldn't remove this from the head includes, the problem should be fixed in the YAML front matter

**Review by dcwalk — COMMENTED** [review]
Okay I think this might be a better strategy @shaqsingh, thoughts?

**Review by shaqsingh — COMMENTED** [review]
looks good!

**Review by dcwalk — APPROVED** [review]
review from @shaqsingh!

**Comment by dcwalk** [comment]
Okay I'll merge in and we can deploy.

**Pull request merged**